src/comments.js

```diff
@@ -897,6 +897,10 @@ function isBlockComment(comment) {
 }
 
 function getCommentChildNodes(node) {
+  // TODO why is this called with node == `null`??
+  if (!node) {
```

Comment:
This is unexpected. https://github.com/prettier/prettier/blob/e22a5516f5b2c44e6d68be4c0a7ba0b32c1b9f86/src/main/comments/attach.js#L43C5-L43C21


Prettier should not pass non-object here.